common/views/components/WorkTags/WorkTags.js

```diff
@@ -0,0 +1,91 @@
+import styled from 'styled-components';
+import { spacing, font, classNames } from '../../../utils/classnames';
+import NextLink from 'next/link';
+import type { NextLinkType } from '@weco/common/model/next-link-type';
+
+export type WorkTagType = {|
+  query: string,
+  textParts: string[],
+  linkAttributes: NextLinkType,
+|};
+
+const WorkTag = styled.div`
+  border-radius: 3px;
+  text-decoration: none;
+  padding: 0.2em 0.5em;
+  transition: color 250ms ease, background 250ms ease;
+`;
+
+type Props = {
+  tags: WorkTagType[],
+};
+
+const WorkTags = ({ tags }: Props) => {
+  return (
+    <div
+      className={classNames({
+        // Cancel out space below individual tags
+        [spacing({ s: -2 }, { margin: ['bottom'] })]: true,
```

Comment:
Where does this come from?

Reply:
Each tag has 12px bottom margin so consecutive rows don't butt up, but we don't know how many tags will be on the final row, so can't just e.g. remove bottom margin from the last child.

I think CSS grid would make this slightly grossness go away – wonder when the right point to jump into grid for stuff like this would be. Maybe this, and anyone without grid just gets the tags stacked up.

Reply:
> I think CSS grid would make this slightly grossness go away

Thinking about it some more, I don't think it does at all. Entirely the wrong tool for the job. Ignore me.